openpype/hosts/houdini/plugins/load/load_image.py

```diff
@@ -42,9 +42,9 @@ def get_image_avalon_container():
 
 
 class ImageLoader(load.LoaderPlugin):
-    """Specific loader of Alembic for the avalon.animation family"""
+    """Load images into COP2"""
 
-    families = ["colorbleed.imagesequence"]
+    families = ["imagesequence"]
```

Comment:
Note: This is not just a cosmetic change. However the `colorbleed.imagesequence` family does not exist in the OpenPype code base and thus this didn't work previously anyway.

It should however be part of the Houdini cleanup and improvements to actually fix these up and test the behaviors as I believe OpenPype actually doesn't use the `imagesequence` family either but instead uses `render` as family. Nonetheless there are still quite a few plug-ins that are set to operate on `imagesequence` instead.

Reply:
Very true.